src/apps/datacopy.c

```diff
@@ -445,11 +445,8 @@ create_target_table(char *sobjname, char *owner, char *dobjname, DBPROCESS * dbs
 
 		strlcat(ls_command, colinfo.ServerTypeDeclaration, sizeof(ls_command));
 
-		if (colinfo.Null == TRUE) {
-			strlcat(ls_command, " NULL", sizeof(ls_command));
-		} else {
-			strlcat(ls_command, " NOT NULL", sizeof(ls_command));
-		}
+		strlcat(ls_command, (colinfo.Identity == TRUE) ? " IDENTITY" : " ", sizeof(ls_command));
+		strlcat(ls_command, (colinfo.Null == TRUE) ? " NULL" : " NOT NULL", sizeof(ls_command));
```

Comment:
Not a regression but I would avoid checking `<bool> == TRUE` and just use `<bool>` or `!!<bool>` if you really want the expression to be bool

Reply:
Will fix. Thanks.

Reply:
@freddy77 any reason why Null is a BYTE and not a BOOL?

Reply:
The definition of the original structure came from Microsoft. Possibly they did this decision to save some bytes.

Reply:
In that case I'd keep the `<bool> == TRUE` for the sake of consistency. What do you think ? Would feel weird to see

```c
strlcat(ls_command, (colinfo.Identity) ? " IDENTITY" : " ", sizeof(ls_command));
strlcat(ls_command, (colinfo.Null == TRUE) ? " NULL" : " NOT NULL", sizeof(ls_command));
```

Also to be honest, I've never liked if(value) because `if` treats *any* no zero value to true. `== TRUE` is safer in that regard.

Reply:
I propose not to change that one for the reasons mentioned above.